src/pages/basic_data/app_admin/central_import/CentralImportDuplicateSearch.tsx

```diff
@@ -52,7 +56,24 @@ export const CentralImportDuplicateSearch = ({ duplicateSearchFilters }: Central
           </Typography>
           {duplicateCandidatesSize > 0 && (
             <>
-              <RegistrationList registrations={duplicateCandidatesQuery.data?.hits ?? []} />
+              <FormControl>
+                <RadioGroup
+                  value={registrationIdentifier}
+                  onChange={(event) => setRegistrationIdentifier(event.target.value)}>
+                  {duplicateCandidatesQuery.data?.hits.map((registration) => (
+                    <FormControlLabel
```

Comment:
Får ikke disse til å ta hele bredden... skjønner ikke hva jeg misser 😩 
![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/ca93064b-2e92-49ae-bad6-066557e9abb9)

Reply:
Nei, denne skjønner jeg meg ikke på heller :thinking:

Reply:
Fant ei løsning på det:
```suggestion
                    <FormControlLabel
                    sx={{ '.MuiFormControlLabel-label': { width: '100%' } }}
```

Reply:
ah fakk, jeg prøvde bl.a. med `.MuiFormControlLabel`, men det var `-label`-postfiksen som gjorde det da ja 🤔 litt hacky, men tenker vi bare får gjør det sånn da 🤷

Reply:
Alle blir like brede, men de gikk fortsatt ikke alltid helt ut da. MEN når jeg la til 100% vredde på FormControl også ble det rett 👍